### PR TITLE
HDRP - Update docs related to FrameSettings, HDRP settings and Default Settings

### PR DESCRIPTION
**Pull request by pmavridis**
### Purpose of this PR
Several settings have been moved, removed or changed, this PR makes sure that doc is up to date.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md [hunk 1]
```diff
@@ -91,6 +92,10 @@ These settings control lighting features for your rendering components. Here you
...
 | **Sky Reflection**                 | Enable the checkbox to allow this Camera to use the Sky Reflection. The Sky Reflection affects specular lighting. |
 | **Direct Specular Lighting**       | Enable the checkbox to allow this Camera to render direct specular lighting. This allows HDRP to disable direct view dependent lighting. It doesn't save any performance. |
+| **Subsurface Scattering** | Enable the checkbox to make HDRP compute Screen Space Subsurface Scattering. |
+| - **Quality Mode**        | Use the drop-down to select how the quality level for Subsurface Scattering is set.<br /> **From Quality Settings**: The Camera uses the Subsurface Scattering **Sample Budget** property that is set in the Material Section of your Unity Project's **Quality Settings**.<br /> **Override Quality Settings**: Allows you to set a custom sample budget for sample surface scattering for this Camera in the field below.|
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md:96]: After the line breaks, can you add a bullet point( &#8226; )? So
<br/> &#8226;

File: com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md [hunk 2]
```diff
@@ -91,6 +92,10 @@ These settings control lighting features for your rendering components. Here you
...
+| - **Quality Level**       | Use the drop-down to select the quality level when the the **Quality Mode** is set to **From Quality Settings**.|
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md:97]: Duplicate "the"

File: com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md [hunk 3]
```diff
@@ -91,6 +92,10 @@ These settings control lighting features for your rendering components. Here you
...
+| - **Custom Sample Budget**       | The custom number of samples that will be used for Subsurface Scattering computations when the **Quality Mode** is set to **Override Quality Settings**.|
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md:98]: The custom number of samples to use for Subsurface Scattering calculations when the **Quality Mode** is set to **Override Quality Settings**.

File: com.unity.render-pipelines.high-definition/Documentation~/HDRP-Asset.md [hunk 4]
```diff
@@ -49,7 +49,7 @@ When you create an HDRP Asset, open it in the Inspector to edit its properties.
...
 | - **Custom Buffer Format**              | Specify the texture format for the custom buffer. If you experience banding issues due to your custom passes, you can change it to either `R11G11B10` if you don't need alpha or `R16G16B16A16`. |
 | **Realtime Raytracing (Preview)**       | Enable the checkbox to enable HDRP realtime ray tracing (Experimental). It requires to have ray tracing compatible hardware. For more information, please refer to the [Ray Tracing Getting Started](Ray-Tracing-Getting-Started.md#HardwareRequirements) page. |
-| **Raytracing Tier**                     | Select the active tier for ray tracing effects. For more information, please refer to the [Ray Tracing Tier Table](Ray-Tracing-Getting-Started.md#TierTable). |
+| **Supported Ray Tracing Mode (Preview)**| Select the supported modes for ray tracing effects (Performance, Quality or Both). For more information, please refer to the [Ray Tracing Tier Table](Ray-Tracing-Getting-Started.md#TierTable). |
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/HDRP-Asset.md:52]: For more information, see the [Ray tracing tier table](Ray-Tracing-Getting-Started.md#TierTable).

File: com.unity.render-pipelines.high-definition/Documentation~/HDRP-Asset.md [hunk 5]
```diff
@@ -67,6 +67,7 @@ These settings control the draw distance and resolution of the decals atlas that
 | **- Atlas Height**                           | The Decal Atlas height. This atlas stores all decals that project onto transparent surfaces. |
 | **- Metal and Ambient Occlusion properties** | Enable the checkbox to allow decals to affect metallic and ambient occlusion Material properties. Enabling this feature has a performance impact. |
 | **- Maximum Clustered Decals on Screen**     | The maximum number of clustered decals that can affect transparent GameObjects on screen. Clustered decals refer to a list of decals that HDRP uses when it renders transparent GameObjects. |
+| **- Layers**                                 | Enable the checkbox to allow decals to affect only specific layers.|
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/HDRP-Asset.md:70]: Enable the checkbox to allow decals to only affect specific layers.

File: com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md (hunk 1 above)
Comment (pmavridis) [reply com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md]: Done! Also it turns out we already had an entry for Subsurface Scattering, but it the wrong place/order and without the extra fields for quality/samples, so I fixed this issue too.

File: com.unity.render-pipelines.high-definition/Documentation~/HDRP-Asset.md (hunk 4 above)
Comment (pmavridis) [reply com.unity.render-pipelines.high-definition/Documentation~/HDRP-Asset.md]: Ah, I missed this one: the target page does not have a tier table anymore, so I changed that to a general  link to the RT page. If you see anything else let me know.

**Pull request merged**